menas/pom.xml

```diff
@@ -164,6 +164,11 @@
             <artifactId>enceladus-migrations</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
I'm pretty sure I've added the dependency in the previous PR. Please, be careful merging this. Rebasing against the current develop might be a good idea.

Reply:
Into menas pom?

Reply:
Yes, I've added it to check if it is compatible.

Reply:
Will check thx